src/main/java/org/commcare/formplayer/services/NewFormResponseFactory.java

```diff
@@ -71,10 +71,11 @@ public NewFormResponse getResponse(NewSessionRequestBean bean, String postUrl) t
                 bean.getSessionData().getAppId(),
                 bean.getRestoreAs(),
                 bean.getRestoreAsCaseId());
+        storageFactory.getStorageManager().registerStorage(FormDef.STORAGE_KEY, FormDef.class);
```

Comment:
What do you think about putting this in a method in `FormplayerStorageFactory` with a javadoc to give a bit more context of why it's needed? It would also tie the two usages together.

Reply:
Yeah anything to make this less confusing in the future sounds good to me. See d82f76ea27cde69ac1bd3b1180d744162ebd9307 and let me know if the javadoc could be more informative.